src/core/include/openvino/op/util/keep_in_graph_op.hpp

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <vector>
+
+#include "openvino/op/util/framework_node.hpp"
+
+namespace ov {
+namespace op {
+namespace util {
+
+class OPENVINO_API KeepInGraphOp : public ov::op::util::FrameworkNode {
```

Comment:
please move this operation into `helper_ops` of tensorflow_common and inherit from `InternalOperation`.

Reply:
Done.